### ISSUE TITLE
literal input component UX improvement

### ISSUE THREAD

**Issue by michelleif** [LD4P/sinopia_editor#3011]
make changes to current literal input component, see Astrid's design below.

here is a list of the changes that are significant, when comparing to current look and behavior:

1. when user enters data, data will remain in input box (currently it gets "entered" and displayed below box) QUESTION: when does the data become part of RDF preview? when you hit enter? tab out of box? ANSWER: When you move out of the box in any way.
2. remove Edit button (no longer needed because user can edit entered value directly in input box)
2. if field is repeatable, show "+Add another" button, which when clicked gives user another "row" to input another value
3. "Language..." button moves to right of input box, to right of diacritic button
4. add trash can to right of Language button on each "row". when user clicks the trash can it removes the row. there will always be at least one row (i.e., when you delete the first and only row, it is replaced with a blank row)
5. if user enters more than one value, add Trash can to right of field name (to remove all entered values)
6. background is white (vs current dark beige / light beige) 
7. add carat to left of field name to allow user to collapse (see additional image below of what collapsed looks like)

**Comment by astridu** [comment]
![image](https://user-images.githubusercontent.com/42216982/133817152-21e3429b-3104-4337-a1a4-b9c81be24764.png)

**Comment by justinlittman** [comment]
Implementation notes: 
* The redux state should not get updated on every change to the input box. Rather:
  * The value should be cached in local state until blur, in which case it should be updated in the Redux state.
  * The Save button should be enabled as soon as a change is made (regardless of blur)
* Check before deviating from standard BS5 components.

**Comment by michelleif** [comment]
when collapsed:
![collapse_literal.png](https://images.zenhubusercontent.com/5d825bf74def500001d12e70/b5a3a3f9-2905-4d41-842b-6f19c8555f23)

**Comment by michelleif** [comment]
note: we are NOT changing the current behavior of:
-required fields get an asterisk and must be completed before Saving
-fields with default value are pre-filled but editable
-fields with a remark or remark URI get tooltip or help link respectively

**Comment by michelleif** [comment]
QUESTION @astridu @justinlittman: what is desired behavior of  trash can to right of field name (the one that gets added once more than one row is filled in). in #5 in original description of ticket I said it  should remove all entered values, but shouldn't it leave one blank row? otherwise user has to re-add a row (see #2845)...need to update lookup (#3012) and URI component (#3013) tickets to explain behavior of field-level trash can
